web/docs/Frontend Starters/Gatsby/Gatsby + WordPress/setting-environment-variables.md

```diff
@@ -20,3 +20,48 @@ For example:
 ```
 WPGRAPHQL_URL=https://my-wordpress-site.pantheon.site/wp/graphql
 ```
+
+## Connecting to Multidev Environments
+
+To connect to a Multidev environment, the following helper environment variable
+can be used inside of `gatsby-config.js`.
+
+- `IS_LIVE_ENVIRONMENT` - True if `PANTHEON_ENVIRONMENT_URL` is live.
+
+Either the `PANTHEON_CMS_ENDPOINT` or `WPGRAPHQL_URL` will need to be set.
+
+The `PANTHEON_CMS_ENDPOINT` can be mocked for local development by defining it
+in the `.env.development.local`.
+
+```
+PANTHEON_CMS_ENDPOINT=dev-my-gatsby-site.pantheonsite.io
+```
+
+Taking a look at how the `gatsby-config.js` works, there is this logic which
+sets the `url`.
+
+```js
+let url = process.env.WPGRAPHQL_URL || process.env.PANTHEON_CMS_ENDPOINT;
+```
+
+In order to connect to a Multidev backend, this url will need to be updated.
+`PANTHEON_ENVIRONMENT_URL` includes the PR number or integration branch name and
+you can parse it to your needs.
+
+This code could be added under the above logic to connect to a Multidev that is
+prefixed with the branch name of my site.
+
+```js
+/**
+* My branch is named `multi-demo`. I will parse the environment url for that substring
+* and use that, along with the `WPGRAPHQL_URL`, to create a `url` which points
+to my Multidev backend.
+**/
+const PREFIX = process.env.PANTHEON_ENVIRONMENT_URL.match(/^([^-]*-)[^-]*/)[0];
+if (!process.env.IS_LIVE_ENVIRONMENT) {
+	url = `https://${PREFIX}-${process.env.WPGRAPHQL_URL.replace(
```

Comment:
```suggestion
	url = `https://${PREFIX}-${url.replace(
```
Two things here, I think 1 also applies to the other docs on this topic:
1. The `WPGRAPHQL_URL` will have the linked/provided multidev prefix like `dev-my-drupal-site.pantheon.site.io` so this url will end up being `https://multi-demo-dev-my-drupal-site.pantheon.site.io` which won't work.
2. Use `url` here over `WPGRAPHQL_URL` because it's possible that `WPGRAPHQL_URL` isn't set but `url` will be either way.

Reply:
To clarify for point 1, since the `WPGRAPHQL_URL` will have the linked/provided multidev prefix,`dev` will will need to be parsed out to make the url end up like `https://multi-demo-my-drupal-site.pantheon.site.io` in order to work. Correct?

Reply:
Yes. Unless I'm missing something, as is it looks like we'll get an invalid URL for the backend.